src/parthenon_manager.cpp

```diff
@@ -75,6 +75,20 @@ ParthenonStatus ParthenonManager::ParthenonInit(int argc, char *argv[]) {
 
   Kokkos::initialize(argc, argv);
 
+  // pgrete: This is a hack to disable allocation tracking until the Kokkos
+  // tools provide a more fine grained control out of the box.
+  auto *env_track_alloc = std::getenv("KOKKOS_TRACK_ALLOC_OFF");
```

Comment:
is an environment variable how we want to keep track of this? We could also put it into `pin` or something?

Reply:
Yes, I think that the most consistent way of action (given that the Kokkos profiling itself is also controlled via setting the `KOKKOS_PROFILE_LIBRARY` variable).
Also, this change comes from a different PR #388